app/src/main/java/org/eyeseetea/malariacare/domain/entity/Option.java

```diff
@@ -0,0 +1,58 @@
+package org.eyeseetea.malariacare.domain.entity;
+
+
+public class Option {
```

Comment:
This domain entity is anemic. This is something @xurxodev has already warned us about (http://xurxodev.com/clean-architecture-code-smells-parte-2/). Can you propose a way to provide this entity with some logic?